content/ja/docs/concepts/storage/projected-volumes.md

```diff
@@ -0,0 +1,91 @@
+---
+title: 投影ボリューム
+content_type: concept
+weight: 21 # just after persistent volumes
+---
+
+<!-- overview -->
+
+このドキュメントでは、Kubernetesの*投影ボリューム*について説明します。[ボリューム](/docs/concepts/storage/volumes/)に精通していることをお勧めします。
+
+<!-- body -->
+
+## 概要
+
+ボリュームは、いくつかの既存の`投影`ボリュームソースを同じディレクトリにマップします。
+
+現在、次のタイプのボリュームソースを投影できます。
+
+* [`secret`](/docs/concepts/storage/volumes/#secret)
+* [`downwardAPI`](/docs/concepts/storage/volumes/#downwardapi)
+* [`configMap`](/docs/concepts/storage/volumes/#configmap)
+* `serviceAccountToken`
+
+すべてのソースは、Podと同じnamespaceにある必要があります。詳細は[all-in-one volume designドキュメント](https://github.com/kubernetes/design-proposals-archive/blob/main/node/all-in-one-volume.md)を参照してください。
+
+### secret、downwardAPI、およびconfigMapを使用した構成例 {#example-configuration-secret-downwardapi-configmap}
+
+{{< codenew file="pods/storage/projected-secret-downwardapi-configmap.yaml" >}}
+
+### 構成例:デフォルト以外のアクセス許可モードが設定されたsecret {#example-configuration-secrets-nondefault-permission-mode}
+
+{{< codenew file="pods/storage/projected-secrets-nondefault-permission-mode.yaml" >}}
+
+各投影ボリュームソースは、specの`sources`にリストされています。パラメータは、2つの例外を除いてほぼ同じです。
+
+* secretについて、ConfigMapの命名と一致するように`secretName`フィールドが`name`に変更されました。
+* `defaultMode`はprojectedレベルでのみ指定でき、各ボリュームソースには指定できません。ただし上に示したように、個々の投影ごとに`mode`を明示的に設定できます。
+
+`TokenRequestProjection`機能が有効になっている場合、現在の[サービスアカウントトークン](/ja/docs/reference/access-authn-authz/authentication/#サービスアカウントトークン)を指定されたパスのPodに挿入できます。例えば:
+
+{{< codenew file="pods/storage/projected-service-account-token.yaml" >}}
+
+例のPodには、挿入されたサービスアカウントトークンを含む投影ボリュームがあります。このトークンはPodのコンテナがKubernetes APIサーバーにアクセスするために使用できます。この`audience`フィールドにはトークンのオーディエンスが含まれています。トークンの受信者は、トークンのオーディエンスで指定された識別子で自分自身を識別する必要があります。そうでない場合はトークンを拒否する必要があります。このフィールドはオプションで、デフォルトではAPIサーバーの識別子が指定されます。
+
+`expirationSeconds`はサービスアカウントトークンが有効であると予想される期間です。
+デフォルトは1時間で、最低でも10分(600秒)でなければなりません。
+管理者は、APIサーバーに`--service-account-max-token-expiration`オプションを指定することで、その最大値を制限することも可能です。
+`path`フィールドは、投影ボリュームのマウントポイントへの相対パスを指定します。
+
+
+{{< note >}}
+投影ボリュームソースを[`subPath`](/docs/concepts/storage/volumes/#using-subpath)ボリュームマウントとして使用しているコンテナは、それらのボリュームソースの更新を受信しません。
+{{< /note >}}
+
+## SecurityContextの相互作用
+
+[サービスアカウントの投影ボリューム拡張でのファイル権限処理の提案](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/2451-service-account-token-volumes#proposal)により、正しい所有者権限が設定された投影ファイルが導入されました。
```

Comment:
I have made change on the upstream.
https://github.com/kubernetes/website/pull/31496/files

```suggestion
サービスアカウントの投影ボリューム拡張でのファイル権限処理の[提案](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/2451-service-account-token-volumes#proposal)により、正しい所有者権限が設定された投影ファイルが導入されました。
```